src/Conf.ml

```diff
@@ -783,12 +767,66 @@ let default =
   ; module_item_spacing= C.default Formatting.module_item_spacing
   ; ocp_indent_compat= C.default Formatting.ocp_indent_compat
   ; parens_tuple= C.default Formatting.parens_tuple
+  ; profile= Fn.id
   ; quiet= C.default quiet
   ; sequence_style= C.default Formatting.sequence_style
   ; type_decl= C.default Formatting.type_decl
   ; wrap_comments= C.default Formatting.wrap_comments
   ; wrap_fun_args= C.default Formatting.wrap_fun_args }
 
+let janestreet_profile =
+  { default with
+    break_cases= `Fit
+  ; break_infix= `Fit_or_vertical
+  ; break_string_literals= `Wrap
+  ; doc_comments= `Before
+  ; extension_sugar= `Preserve
+  ; field_space= `Loose
+  ; if_then_else= `Keyword_first
+  ; indicate_nested_or_patterns= false
+  ; infix_precedence= `Parens
+  ; leading_nested_match_parens= true
+  ; let_and= `Sparse
+  ; let_open= `Preserve
+  ; margin= 90
+  ; ocp_indent_compat= true
+  ; parens_tuple= `Multi_line_only
+  ; sequence_style= `Terminator
+  ; type_decl= `Sparse
+  ; wrap_fun_args= false
+  ; module_item_spacing= `Compact }
+
+let profile =
+  let doc = "Preset profiles." in
+  let names = ["p"; "profile"] in
+  let all =
+    [ ("none", Fn.id, "$(b,none) when no profile is selected.")
```

Comment:
What's the purpose of this one ?

Reply:
I think we need a profile that does nothing that will be the default value of the option, so that we do not override the options that are already set (since the "default" profile sets each option now) even when no profile is set.

Reply:
Sorry, I don't follow. As far as I understand, `default` is the default value for that option because this is what we start with while build the config.

Reply:
@gpetiot do you think that we need a do-nothing profile, or could we just not specify one in such use cases?